autoscheduler/frontend/src/components/LandingPage/SelectTerm.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import {
+  Menu, MenuItem, Grid, Button, Typography
+} from '@material-ui/core';
+
+const options = [
+  'None',
+  'Semester 1',
+  'Semester 2',
+  'Semester 3',
+  'Semester 4',
+  'Semester 5',
+  'Semester 6',
+];
+
+const ITEM_HEIGHT = 48;
+
+const SelectTerm: React.SFC = function App() {
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+
+  const handleClick = (event: React.MouseEvent<HTMLElement>): void => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = (): void => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <div>
+      <Grid
+        container
+        spacing={0}
+        direction="column"
+        justify="space-around"
+        alignItems="center"
+      >
+        <Button
+          style={{ width: '55%', maxWidth: '800px' }}
+          variant="contained"
+          color="secondary"
+          aria-controls={open ? 'menu-list-grow' : undefined}
+          aria-haspopup="true"
+          onClick={handleClick}
+        >
+            Select Term
+        </Button>
+        <Menu
+          id="long-menu"
+          anchorEl={anchorEl}
```

Comment:
I'm getting this error in the console, and I think it's related to this line. Any idea how to fix it? If not, I can try to help:
```Material-UI: you can not change the default `anchorOrigin.vertical` value when also providing the `get Content AnchorEl` prop to the popover component. Only use one of the two props. Set `getContent AnchorEl` to `null | undefined` or leave `anchorOrigin.vertical` unchanged.```

Reply:
I'm trying to create the anchor position to the button and then center it on that position so everything lines up. But you cannot set the horizontal position without setting the vertical which is what it appears to be getting mad at. And if you don't have either it looks bad.

Reply:
Fixed in #107